model/item.cc

```diff
@@ -248,8 +286,46 @@ namespace minsky
   }
 
   void BottomRightResizerItem::drawResizeHandles(cairo_t* cairo) const
-  {
-    drawResizeHandle(cairo,right()-x(),bottom()-y(),0.5*resizeHandleSize(),0);
+  { 
+    // ensure resize handle is always drawn on the same corner of variable/items for 90 and 180 degree rotations. for ticket 1232   
+    double fm=std::fmod(rotation(),360), angle;	
+    float x1=right(),y1=bottom();  
+    if (fm==-90 || fm==270) {
+      angle=-rotation();
+      Rotate r1(angle,this->x(),this->y());
+      x1=r1.x(right(),bottom());
+      y1=r1.y(right(),bottom());						  
+    }
+    else if (abs(fm)==180) {
+      angle=rotation();
+      x1=right();
+      y1=top();					
```

Comment:
It strikes me there is a lot of shared code between onResizeHandle and drawResizeHandle that can be DRY'd.